docs/src/pages/components/masonry/ImageMasonry.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry columns={3} spacing={1} sx={{ height: 300 }}>
```

Comment:
```suggestion
    <Masonry columns={3} spacing={1} sx={{ height: 400 }}>
```

to be able to see all of at least one portrait image